lint/defaultCaseOrder_checker.go

```diff
@@ -0,0 +1,56 @@
+package lint
+
+//! Detects when default case in switch isn't on 1st or last position.
+//
+// @Before:
+// switch {
+// case x > y:
+// 	// ...
+// default: // <- not the best position
+//	// ...
+// case x == 10:
+//	// ...
+// }
+//
+// @After:
+// switch {
+// case x > y:
+// 	// ...
+// case x == 10:
+//	// ...
+// default: // <- everything is good
+//	// ...
+// }
+
+import "go/ast"
+
+func init() {
+	addChecker(&defaultCaseOrderChecker{}, attrSyntaxOnly)
```

Comment:
Do we want to make it stable right away?
If not, we should probably add `attrExperimental` here.